src/platforms/unity/common/troubleshooting.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title:  Troubleshooting
+sidebar_order: 100
+description: "Solutions to common issues"
+---
+
+## Installation Issues
+
+### Installation via Git URL - `'HEAD': cannot update ref 'refs/heads/master'`
+
+Some Unity versions, such as `2019.4.24f1` and `2020.3.2f1`, have a bug on UPM and fail to install with the error `'HEAD': cannot update ref 'refs/heads/master'`.
+Unity has resolved this issue on newer releases. Learn more by checking the [Unity Issue Tracker](https://issuetracker.unity3d.com/issues/package-resolution-error-when-using-a-git-dependency-referencing-an-annotated-tag-in-its-git-url).
+
+### PrecompiledAssemblyException: `Multiple precompiled assemblies with the same name`
```

Comment:
Another one to add is:

![image](https://user-images.githubusercontent.com/1633368/124510726-76a69080-dda2-11eb-8c9e-35731a29f4f9.png)


Basically add an empty line to the end it and it'll say: "Couldn't find it.." or something

Reply:
Good point! I'll look up the exact error message to put there too!